### ISSUE TITLE
STB Reason: can't fopen

### ISSUE THREAD

**Issue by dexception** [AlexeyAB/darknet#774]
Loading weights from darknet53.conv.74...TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH TH 
 seen 64 
Done!
Learning Rate: 0.001, Momentum: 0.9, Decay: 0.0005
Resizing
576
Cannot load image "/data/obj/000000295628.jpg"
STB Reason: can't fopen
Cannot load image "/data/obj/000000572405.jpg"
STB Reason: can't fopen
Cannot load image "/data/obj/000000236381.jpg"
STB Reason: can't fopen
Cannot load image "/data/obj/000000253432.jpg"
STB Reason: can't fopen
Cannot load image "/data/obj/000000461998.jpg"
STB Reason: can't fopen
Cannot load image "/data/obj/000000195267.jpg"
STB Reason: can't fopen
Cannot load image "/data/obj/000000326433.jpg"
STB Reason: can't fopen
^C^X^Z^X^C^CSegmentation fault (core dumped)

The image exists and is not corrupted. I have checked this multiple times.

**Comment by dexception** [comment]
- Replaced \r\n with Unix file endings \n
- Changed /data/obj/000000326433.jpg with data/obj/000000326433.jpg. Removed slash in the begining.


*Fixed

**Comment by AlexeyAB** [comment]
Yes, if you prepared `train.txt` on Windows and then you it on Linux/Unix, then you should do in the console:
```
sudo apt-get install dos2unix
dos2unix train.txt
```
